app/services/CheckoutService.scala

```diff
@@ -389,32 +390,63 @@ class CheckoutService(
       subscriptionDetails.withContextLogging(s"Subscription price:")
     }
 
-    def pricesMatch(displayedPrice: String, subscriptionPrice: String) = {
-      if(displayedPrice != subscriptionPrice){
-        logger.error(s"Client and server side prices divergent, we showed $displayedPrice and wanted to charge $subscriptionPrice from ${subscription.name} (${renewal.promoCode})")
-        false
+    case class ContactInfo(billto: ZuoraContact, soldto: ZuoraContact, salesforceContact: Contact)
+
+    def postRenewalSteps(subscription: Subscription[WeeklyPlanOneOff], renewal: model.Renewal, subscriptionPrice: String, salesforceContact: Contact, newTermStartDate: LocalDate) = {
+      def logException(e: Exception): Unit = {
+        logger.error(s"unexpected error in post renewal steps", e)
```

Comment:
Could also explicitly state what has failed (i.e. sub has been renewed but email has not been sent) so we know exactly what this means from Sentry without digging into the code.